src/core-sdk-tasks/GenerateMSBuildExtensionsSWR.cs

```diff
@@ -55,7 +58,16 @@ private void AddFolder(StringBuilder sb, string relativeSourcePath, string swrIn
                 {
                     sb.Append(@"  file source=""$(PkgVS_Redist_Common_Net_Core_SDK_MSBuildExtensions)\");
                     sb.Append(Path.Combine(relativeSourcePath, Path.GetFileName(file)));
-                    sb.AppendLine("\"");
+                    sb.Append('"');
```

Comment:
Do you need to control the NGEN priority? I believe it should default to 3 (it starts when the machine is idle), but you can override it which could trigger NGEN to happen sooner.

Reply:
Priority 3 should be fine. MSBuild itself uses pri 1 for some assemblies and the default for others. These assemblies fall into the not-so-critical category I believe.